server/helpers/dateHelper/index.js

```diff
@@ -0,0 +1,28 @@
+const moment = require("moment");
+
+/**
+ * returns the date of one week/two weeks/ or three weeks from our current date.
+ * @param {integer} numberOfWeeks number of weeks to add to current date.
+ * @returns {date} date object representing the target date based on number of weeks.
+ */
+
+const getTargetDate = (numberOfWeeks) => {
+  // input validation
+  if (typeof numberOfWeeks !== "number") {
+    throw new Error("dateHelper/index.js: number of weeks must be a number");
```

Comment:
Since we are using `boom`  everywhere I think this should be like this. (this goes for everywhere `new Error('bla bla')` is used
@MohammedYehia  what do you think?

```suggestion
    throw boom.badData("dateHelper/index.js: number of weeks must be a number");
```

Reply:
True.

Reply:
I agree 👍

Reply:
do you need to show this message to clients?
**dateHelper/index.js:** number of weeks must be a number

Reply:
@FarahZaqout

Reply:
No. It's a developer error. This part of the code does not interact with any react components.